src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.spec.ts

```diff
@@ -66,9 +72,15 @@ describe('ContentTypeFieldsAddRowComponent', () => {
         comp.rowState = 'add';
         fixture.detectChanges();
         const addRowContainer = de.query(By.css('.dot-add-rows-button__container'));
-        const buttonElement = de.query(By.css('button'));
+        const buttonsElement = de.queryAll(By.css('button'));
         expect(addRowContainer.nativeElement.classList.contains('dot-add-rows__add')).toEqual(true);
-        expect(buttonElement).toBeTruthy();
+        expect(buttonsElement[0].nativeElement.textContent).toBe('Add Row');
```

Comment:
When you do this types of assertions over the DOM element of a primeng component is like you testing their component, you should only test that the attr and params you passing to that component are correct, it gets less complicated.

Reply:
?